.env.example

```diff
@@ -19,7 +19,7 @@ BROADCAST_DRIVER=log
 CACHE_DRIVER=file
 FILESYSTEM_DRIVER=local
 QUEUE_CONNECTION=sync
-SESSION_DRIVER=file
```

Comment:
no need to include on the PR, please remove

Reply:
How can I fix it?